integration-tests/package.json

```
@@ -4,8 +4,8 @@
   "version": "0.5.22",
   "description": "[Optimism] Integration tests",
   "scripts": {
-    "lint": "yarn lint:fix && yarn lint:check",
-    "lint:fix": "yarn lint:check --fix",
+    "lint": "pnpm lint:fix && pnpm lint:check",
```

Unrelated to this pr but this is not a normal way of doing things.   CI should be using lint:check and this has no reason to be running the linter twice

```suggestion
    "lint": "pnpm lint:fix",
```

Don't commit this as this is out of scope for pr